src/daemon.js

```diff
@@ -131,6 +131,9 @@ class Daemon {
       }
 
       const daemon = execa(this._binPath, execOptions)
+      if (process.env.DEBUG) {
+        daemon.stderr.pipe(process.stderr)
```

Comment:
Aren't library logs suppose to be in process.stdout?

Might be useful to pass custom env to execa, I think something like `execa(this._binPath, execOptions, {env: {DEBUG: "libp2p:noise, libp2p:secio"}})` should be supported

Reply:
All debug logs are going to stderr as we're just using the defaults in the debug module atm.

`execa` forwards env to the new process, so it shouldn't be necessary as we only need forwarding right now. We don't need the conditional though.